pkg/keystone/bootstrap.go

```diff
@@ -32,6 +31,51 @@ func BootstrapJob(cr *keystonev1beta1.KeystoneAPI, configMapName string, APIEndp
 				Spec: corev1.PodSpec{
 					RestartPolicy:      "OnFailure",
 					ServiceAccountName: "keystone",
+
+					InitContainers: []corev1.Container{
```

Comment:
It looks like all 3 uses of this init container are identical. This should be common code.